integration-cli/daemon.go

```diff
@@ -527,3 +527,76 @@ func (d *Daemon) checkActiveContainerCount(c *check.C) (interface{}, check.Comme
 	}
 	return len(strings.Split(strings.TrimSpace(out), "\n")), check.Commentf("output: %q", string(out))
 }
+
+func (d *Daemon) checkRunningTaskCount(c *check.C) (interface{}, check.CommentInterface) {
+	// get service list
+	serviceOutput, err := d.Cmd("service", "ls", "-q")
```

Comment:
I think it would likely be better to hit the tasks API endpoint and iterate over the tasks directly.